app/views/clinicfinders/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+  <div class="col-sm-6">
+    <div class="info-form-left">
+      <div> Zip code </div>
```

Comment:
If it's easier, you can use `bootstrap_form_for url: clinicfinder_path`, and that way you can take advantage of the very straightforward rails helpers that'll generate all the boostrap fluff for you. So something like:

```
<%= bootstrap_form_for url: clinicfinder_path do |f| %>
  <%= f.text_field :zip_code %>
  <%= f.select :approx_gestation_at_appt, weeks_options, label: 'Approx. gestation at appt' %>
<% end %>
```

Might have to put that in divs and rows and junk, but will definitely let you skip some of the scruff!